docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started with Scalar DB on Cosmos DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Cosmos DB with a simple electronic money application.
+
+## Install prerequisites
+
+Scalar DB is written in Java. So the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+
+## Cosmos DB setup
+You also need to configure a Cosmos DB account to get started with Scalar DB on Cosmos DB.
+
+* [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
+    * Select **Azure Cosmos DB** service from the services on Azure web console.
+    * Select **Add**
+    * On the **Create Azure Cosmos DB Account** page, enter the basic settings for the new **Azure Cosmos DB** account.
+        * Create new or choose the existing **Resource Group**
+        * Enter the Cosmos DB **Account Name**
+        * Choose **API** as `Core (SQL)`
+        * Choose **Location**
+        * Select **Review + create**. You can skip the **Network** and **Tags** sections.
+        * Review the account settings, and then select **Create**.
+        *  Wait some time for **Azure Cosmos DB** account creation.
+    * Select **Go to resource** to go to the Azure Cosmos DB account page.
+    * Select **Default consistency** from the left navigation on your Azure Cosmos DB account page,
+        * Change `Consistency Level` from `SESSION` to `STRONG`.
+        * Select **Save**
```

Comment:
```suggestion
This section explains how to set up [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) with Azure portal.

1. Select **Azure Cosmos DB** service from the services on Azure web console.
1. Select **Add**
1. On the **Create Azure Cosmos DB Account** page, enter the basic settings for the new **Azure Cosmos DB** account.
    * Create new or choose the existing **Resource Group**
    * Enter the Cosmos DB **Account Name**
    * Choose **API** as `Core (SQL)`
    * Choose **Location**
    * Select **Review + create**. You can configure the **Network** and **Tags** sections for your environment.
    * Review the account settings, and then select **Create**.
    *  Wait some time for **Azure Cosmos DB** account creation.
1. Select **Go to resource** to go to the Azure Cosmos DB account page.
1. Select **Default consistency** from the left navigation on your Azure Cosmos DB account page,
    * Change `Consistency Level` from `SESSION` to `STRONG`.
    * Select **Save**
```

Reply:
Thank you for the suggestion.
Document updated with the suggestion.